misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -89,5 +92,29 @@ abstract class PathAssertionsSpec(
             }
         }
     }
+
+    describeFun(fileNameWithoutExtension.name) {
+        val fileNameWithoutExtensionFun = fileNameWithoutExtension.lambda
+
+        context("path with extension") {
+            it("does not throw") {
+                expect(Paths.get("a/my.txt")).fileNameWithoutExtensionFun { toBe("my") }
+            }
+
+            it("throws an AssertionError") {
+                expect {
+                    expect(Paths.get("a/my.txt")).fileNameWithoutExtensionFun { toBe("my.txt") }
+                }.toThrow<AssertionError> {
+                    messageContains(DescriptionPathAssertion.FILE_NAME_WITHOUT_EXTENSION.getDefault())
+                }
+            }
+        }
+
+        context("path with double extension") {
+            it("does not throw") {
+                expect(Paths.get("a/my.tar.gz")).fileNameWithoutExtensionFun { toBe("my.tar") }
+            }
+        }
```

Comment:
Should I even include this?

Reply:
Yes please, good test case :+1:

Reply:
Moving this to the feature

Reply:
It's still here :slightly_smiling_face:

Reply:
should be good now :)  decreased the files changed to 9 (this and the subclass)